src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -1644,8 +1645,12 @@ class LineChart extends TemporalXAxisComponent {
                                 yExtent = self.swimlaneYExtents[agg.swimLane];
                             }
 
-                            // Update yExtent in LineChartData after all yExtent updates (this is public facing yExtent)
-                            self.chartComponentData.setYExtents(i, yExtent);
+                            // Update yExtent key in LineChartData after all local yExtent updates (this is public facing yExtent)
```

Comment:
Did you mean update yExtend 'index'?

Reply:
Good catch, that was an object before, I'll make that change